pkg/seachest/seachest.go

```diff
@@ -165,6 +166,18 @@ func (I *Identifier) GetRotationRate(driveInfo *C.driveInformationSAS_SATA) uint
 	return 0
 }
 
+func (I *Identifier) GetRotationalLatency(driveInfo *C.driveInformationSAS_SATA) float64 {
+	if driveInfo.rotationRate > 1 {
+		// latency = 1/rpm/60
+		klog.Infof("Device is : %v", I.DevPath)
+		rpm := (uint16)(driveInfo.rotationRate)
```

Comment:
Provide a reference to this, so that we can see why the formula was chosen

Reply:
@harshthakur9030  Is this going to be the formula we are going to use?. Because still we dont have SSDs handled here.

Reply:
SSD won't have a rotation rate. If it's 0, users can understand that it's an SSD.

Reply:
Thats fine. But what will be the value shown when a user fetches the metrics. Will it be shown as zero, or some other random float value.

Reply:
per the SCSI spec, medium rotation rate = 0 means it is not reported. medium rotation rate = 1 means non-rotating medium (eg solid state)

Reply:
@richardelling Iam a little confused

>  medium rotation rate = 1 means non-rotating medium (eg solid state)

Does rotation rate =1 means the device is an SSD?

Reply:
@akhilerm as far as I understand, if we have an SSD, the latency reported will be zero. In the help section of the exported metrics, we've stated this part. 

@richardelling as I am only working with greater than 1, this code would be displaying latency for HDD, right?

Reply:
correct